src/components/ScoreBoard/ScoreBoard.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from "react";
+
+import Header from "./Header/Header";
+import Player from "./Player/Player";
+
+import './scoreboard.css';
+
+class ScoreBoard extends Component {
+  state = {
+    players: [
+      {
+        name: "Guil",
+        id: 1
+      },
+      {
+        name: "Treasure",
+        id: 2
+      },
+      {
+        name: "Ashley",
+        id: 3
+      },
+      {
+        name: "James",
+        id: 4
+      }
+    ]
+  };
+
+  handleRemovePlayer = (id) => {
+    this.setState( prevState => {
+      return {
+        players: prevState.players.filter(p => p.id !== id)
```

Comment:
Let's consider using a more meaningful name for our variable. The `p` can be replaced by a `player`.